.github/workflows/release.yml

```
@@ -180,13 +180,28 @@ jobs:
           repository: smallstep/docs
           token: ${{ secrets.DOCS_PAT }}
           path: './docs'
+      - name: Setup bot SSH signing key
+        uses: webfactory/ssh-agent@v0.5.3
+        env:
+          has_SSH_PRIVATE_KEY: ${{ secrets.STEP_TRAVIS_CI_GH_PRIVATE_SIGNING_KEY != '' }}
+        if: ${{ env.has_SSH_PRIVATE_KEY != '' }}
```

Should it be 
```suggestion
        env:
          HAS_SSH_PRIVATE_KEY: ${{ secrets.STEP_TRAVIS_CI_GH_PRIVATE_SIGNING_KEY != '' }}
        if: ${{ env.HAS_SSH_PRIVATE_KEY != '' }}
```

---

Or can you do the if with the secret?
```suggestion
        if: ${{ secrets.STEP_TRAVIS_CI_GH_PRIVATE_SIGNING_KEY != '' }}
```

---

Unfortunately you can't use secrets in conditionals. But the code is wrong. I should be comparing to `true`.